core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210615_sharded_entity_tables.xml

```diff
@@ -3,38 +3,33 @@
 
     <!-- create db functions and stored procedures that will help with the db migration -->
     <changeSet logicalFilePath="dummy" author="davidan" id="create_shard_functions_and_procedures">
-        <sql stripComments="true" endDelimiter="/">
+        <sql stripComments="true" endDelimiter="~">
```

Comment:
because I needed to use `/` as the division operator when calculating the lower bound, I needed to change the delimiter